python/src/detector.cpp

```diff
@@ -30,31 +30,31 @@ void init_det(py::module &m) {
         .def(py::init<int>())
 
         .def("freeSharedMemory",
-             (void (Detector::*)()) & Detector::freeSharedMemory)
+             (void(Detector::*)()) & Detector::freeSharedMemory)
```

Comment:
On which PC did you run the generation scripts? Looks like clang-format removed a space. Maybe we can fix this to avoid a huge diff?

Reply:
Pushed a new version without the white space diff.